### PR TITLE
#1-4Task. Vladimir M.

### PR DESCRIPTION
**Pull request by whyauronning**
https://travis-ci.org/github/whyauronning/js-assignments/builds/704699042

### CODE REVIEW

File: task/02-numbers-tasks.js [hunk 1]
```diff
@@ -38,7 +37,8 @@ function getRectangleArea(width, height) {
  *   0    => 0
  */
 function getCicleCircumference(radius) {
-    throw new Error('Not implemented');
+    let p = 3.141592653589793238462643;
```
Comment (FedorovD) [inline task/02-numbers-tasks.js:40]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/PI

File: task/04-arrays-tasks.js [hunk 2]
```diff
@@ -320,9 +400,13 @@ function getPositivesCount(arr) {
  *   [ 'nine','eight','nine','eight'] => [ 'eight','eight','nine','nine']
  *   [ 'one','one','one','zero' ]     => [ 'zero','one','one','one' ]
  */
-function sortDigitNamesByNumericOrder(arr) {
-   throw new Error('Not implemented');
-}
+
+   function sortDigitNamesByNumericOrder(arr) {
+      return arr.sort((a, b) => {
+         var value = {'zero' :0, 'one': 1, 'two': 2, 'three': 3, 'four': 4, 'five': 5, 'six': 6, 'seven': 7, 'eight': 8, 'nine': 9};
```
Comment (FedorovD) [inline task/04-arrays-tasks.js:406]: `var` is deprecated. It has no sense to use for browsers code even currently.

**Comment by FedorovD** [comment]
Closing the PR as accepted! Thanks!

File: task/02-numbers-tasks.js (hunk 1 above)
Comment (whyauronning) [reply task/02-numbers-tasks.js:40]: i got it :) 
thx a lot

File: task/04-arrays-tasks.js (hunk 2 above)
Comment (whyauronning) [reply task/04-arrays-tasks.js:406]: my bad)

**Pull request closed without merging**